mslice/plotting/plot_window/plot_options.py

```diff
@@ -2,13 +2,31 @@
 
 from .plot_options_ui import Ui_Dialog
 
+
 class PlotOptionsDialog(QtGui.QDialog, Ui_Dialog):
     def __init__(self, current_config):
         super(PlotOptionsDialog, self).__init__()
         self.setupUi(self)
-        self.lneFigureTitle.setText(current_config.title)
-        self.lneXAxisLabel.setText(current_config.xlabel)
-        self.lneYAxisLabel.setText(current_config.ylabel)
+        if self.lneFigureTitle is not None:
```

Comment:
Is there a reason that we need these `is not None` checks. I think we can safely assume that the Qt designer has done its job here